lib/ansible/module_utils/distro/_distro.py

```diff
@@ -129,7 +129,9 @@ class InfoDict(TypedDict):
 # Base file names to be looked up for if _UNIXCONFDIR is not readable.
 _DISTRO_RELEASE_BASENAMES = [
     "SuSE-release",
+    "altlinux-release",
     "arch-release",
+    "armbian-release",
```

Comment:
this does not seem to be in the 1.9.0 distro, where did it come from?

Reply:
We keep a copy of distro file from https://github.com/python-distro/distro/blob/master/src/distro/distro.py, this is where armbian changes are coming in.

Reply:
but that is not part of the 1.9.0 release, which is what this PR is stated to do

Reply:
I think I got the issue, we should not take it from the master but from the tag version 1.9.0. I'll push the changes now.